### PR TITLE
Added urbandictionary searching bot command

### PR DESCRIPTION
**Pull request by danzilla**
Hello!

Someone in a discord channel I frequent asked for an urban dictionary searching command so I slapped one together (who knew urbandictionary had an api?)

I copied the style of the !levels and !reddit plugins wrt structure and using the async tools, so hopefully this follows all the rules

### CODE REVIEW

File: chat-bot/plugins/urbandictionary.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            term
+        )
+
+        try:
+            with aiohttp.ClientSession() as session:
+                async with session.get(url) as resp:
+                    if resp.status == 200:
+                        json = await resp.json()
+
+                        response = "No results for {}, you slag".format(
```
Comment (sammyvsparks) [inline chat-bot/plugins/urbandictionary.py]: Probably needs changed, sounds to aggressive

**Comment by danzilla** [comment]
Sorry, still had it tuned for my server's crowd..!

Toned the messages down :)

File: chat-bot/plugins/urbandictionary.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+                'description': 'Looks up given term on urbandictionary.'
+            }
+        ]
+        return commands
+
+    async def on_message(self, message):
+        if message.author.id == self.mee6.user.id:
+            return
+
+        if message.content == '!urbandictionary':
```
Comment (cookkkie) [inline chat-bot/plugins/urbandictionary.py]: Wrong command checking.

File: chat-bot/plugins/urbandictionary.py [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                message.clean_content
+            ))
+
+        term = message.content.split(' ', 1)[1]
+
+        url = "http://api.urbandictionary.com/v0/define?term={}".format(
+            term
+        )
+
+        try:
```
Comment (cookkkie) [inline chat-bot/plugins/urbandictionary.py]: Here you'll basically do that request at every received message

Comment (danzilla) [inline chat-bot/plugins/urbandictionary.py]: Noticed that as well, fixed in recent commit (as well as other point), also url-escaped the supplied term string for extra safety

Comment (danzilla) [inline chat-bot/plugins/urbandictionary.py]: *nudge*

**Comment by BLUUUUUE** [comment]
@danzilla you can close this pull :cookie: is working on it now

**Pull request closed without merging**